production/helm/loki/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: "v1"
 name: loki
-version: 0.24.0
+version: 0.25.0
```

Comment:
you also need to bump loki-chart, as loki chart is part of it.

Reply:
"version: 0.25.0" ist the chart version.
"appVersion: v1.3.0" is the loki version.

or are you talking about loki-stack?

Reply:
> or are you talking about loki-stack?
yes !

Reply:
So i should change the requirements.yaml? 
https://github.com/grafana/loki/blob/master/production/helm/loki-stack/requirements.yaml#L5
It already uses "^0.6.0" which also fits for "0.25.0".

Reply:
no just the version of the chart.yaml

Reply:
OK, raised loki-stack to 0.29.0, even if it feels a bit odd to raise the chart version while no changes to the chart were made.

Imho it would be cleaner to add the explicit chart numbers in the requirements.yaml too.
So the chart version would change when there are changes to requirements.yaml too.